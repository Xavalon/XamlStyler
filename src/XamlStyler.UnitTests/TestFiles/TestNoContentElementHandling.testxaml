<Root>

<!-- START: Test for NO-CONTENT ELEMENT shrink -->

<HasValue>asdf</HasValue>
<HasMixedValue>asdf<child/><!-- comments --></HasMixedValue>
<HasNoValue></HasNoValue>
<HasNoValue a="a" b="b" c="c"></HasNoValue>
<HasEmptyCData><![CDATA[]]></HasEmptyCData>
<HasCData><![CDATA[Test]]></HasCData>
<HasWhitespaceValue>  </HasWhitespaceValue>
<HasMultilineWhitespaceValue>
  
</HasMultilineWhitespaceValue>
<EmptyElement/>

<!-- END: Test for NO-CONTENT ELEMENT shrink -->  

</Root>