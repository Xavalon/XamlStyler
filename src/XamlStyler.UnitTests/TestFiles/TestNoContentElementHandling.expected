<Root>

  <!--  START: Test for NO-CONTENT ELEMENT shrink  -->

  <HasValue>asdf</HasValue>
  <HasMixedValue>
    asdf
    <child />
    <!--  comments  -->
  </HasMixedValue>
  <HasNoValue />
  <HasNoValue a="a"
              b="b"
              c="c" />
  <HasEmptyCData><![CDATA[]]></HasEmptyCData>
  <HasCData><![CDATA[Test]]></HasCData>
  <HasWhitespaceValue />
  <HasMultilineWhitespaceValue />
  <EmptyElement />

  <!--  END: Test for NO-CONTENT ELEMENT shrink  -->

</Root>